cpp/Types.hpp

```diff
@@ -12,7 +12,7 @@ namespace hsd
         template <>
         struct remove_unsigned<unsigned char>
         {
-            using type = char;
+            using type = signed char;
```

Comment:
I don't know why, but ok

Reply:
because if we want it to behave in the same manner as all the other specializations, and the fact that we're using this to define `isize`, I think `signed char` instead of `char` will be more consistent here